models/cam-fv/model_mod.f90

```diff
@@ -3691,6 +3699,9 @@ function above_ramp_start(test_value, gc, obs_type, ramp_end, total_dist, extra_
 ! FIXME: test this!!!
 ! is it above the ramp end? set damp dist to something
 ! large enough to turn off all impacts.  is vert_localize_dist enough?
+! KDR it's 0.3 in my tests on 2021-9-5.
+!     cutoff*vert_norm_scaleheight = 0.15*1.5 = .225, so "yes" in this case.
+! ? Are those values used to set the maxdist in the gc?
 vert_localize_dist = get_maxdist(gc, obs_type)
 if (.false. .and. onetime) then
    print *, 'vert_localize_dist = ', vert_localize_dist
```

Comment:
this debugging code shouldn't have been in the committed version either.  that's my fault.